demo/scenario/script

```diff
@@ -0,0 +1,29 @@
+# Copyright Intel Corp All Rights Reserved
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#  Scripted scenario
+#  (to be excuted by demo/client/scripting/scenario-run.sh)
+#------------------------------------------------------------
+
+scenario
```

Comment:
Maybe rename `/demo/scenario` -> `/demo/scenarios` and thus give some space for different scenarios. Jeb would work a `live demo` scenario, whereas bruno might need a `testing` scenario. `short_video`, `long_video`, can we get the FCC traces and put it in `the_real_fcc_auction`?

Reply:
the script is certainly designed to have different scenarios (sequence of transactions) and different scripting of a particular scenario (more or less of the transactions are through UI).  if you just have different scripts for same transactions, i would have kept it in the same directory. so long and short might be according to my terminology same scenario. FCC trace certainly would be a different one and require a different (sub-)directory.  But i think we can do that once we get there .... :-)